recipes/urwid/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.3.1" %}
+
+package:
+  name: urwid
+  version: "{{ version }}"
+
+source:
+  fn: urwid-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/u/urwid/urwid-{{ version }}.tar.gz
+  md5: 2e1a005cb31368fe21bfeba2d6ad5a5c
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
It sort of seems like the example meta.yaml should contain all of these options and explanations as comments.  That way, if you look at the example or just copy it over, it sort of serves as a walkthrough.